percona/controllers/pgc/pgc.go

```diff
@@ -230,15 +248,15 @@ func (c *Controller) waitForShutdown(stopCh <-chan struct{}) {
 	log.Debug("perconapgcluster Contoller: received stop signal, worker queue told to shutdown")
 }
 
-func (c *Controller) reconcileStatuses(stopCh <-chan struct{}) {
+func (c *Controller) reconcilePerconaPG(stopCh <-chan struct{}) {
 	for {
 		select {
 		case <-stopCh:
 			return
 		default:
-			err := c.handleStatuses()
+			err := c.reconcilePerconaPGClusters()
 			if err != nil {
-				log.Error(errors.Wrap(err, "handle statuses"))
+				log.Error(errors.Wrap(err, "reconcile perocnapgclusters"))
```

Comment:
why we're continuing execution if error?

Reply:
we don't need to exit this function because its kind of reconcile loop